### ISSUE TITLE
Metallb doesn't send/receive ARP when Istio is enabled

### ISSUE THREAD

**Issue by vijaykumark** [google/metallb#330]
**Is this a bug report or a feature request?**:

Bug. Metallb doesn't send or receive ARP requests when Istio is enabled.

**What happened**:
Installed Istio 1.0.2 on Kubernetes v1.11.2, CoreOS. After that installed metallb v0.7.3 and added configuration.
Add loadbalancer ip to a service and it shows that the service got the loadbalancer IP.

There are no logs which say "got ARP request for service IP" as shown below:
```
speaker-5w68n speaker {"caller":"arp.go:102","interface":"eno50","ip":"10.10.2.47","msg":"got ARP request for service IP, sending response","responseMAC":"52:73:33:a6:be:1a","senderIP":"10.10.2.47","senderMAC":"52:73:33:a6:be:1a","ts":"2018-10-17T11:49:26.332788288Z"}
```

**What you expected to happen**:

Metallb should send and receive ARP requests and loadbalancerIP is working fine. Istio should not cause any problems to metallb.

**How to reproduce it (as minimally and precisely as possible)**:
Install k8s v1.11.2 and install Istio v1.0.2 on CoreOS 

**Anything else we need to know?**:

We are running kubernetes on CoreOS on bare metal cluster.

**Environment**:
- MetalLB version: v0.73
- Kubernetes version: v1.11.2
- BGP router type/version: layer2
- OS (e.g. from /etc/os-release):
NAME="Container Linux by CoreOS"
ID=coreos
VERSION=1855.4.0
VERSION_ID=1855.4.0
BUILD_ID=2018-09-11-0003
PRETTY_NAME="Container Linux by CoreOS 1855.4.0 (Rhyolite)"
ANSI_COLOR="38;5;75"
HOME_URL="https://coreos.com/"
BUG_REPORT_URL="https://issues.coreos.com"
COREOS_BOARD="amd64-usr"

- Kernel (e.g. `uname -a`):
Linux system 4.14.67-coreos #1 SMP Mon Sep 10 23:14:26 UTC 2018 x86_64 Intel(R) Xeon(R) CPU E5-2665 0 @ 2.40GHz GenuineIntel GNU/Linux

**Comment by vijaykumark** [comment]
@danderson  Any clue how to fix this issue? 
I have another cluster without Istio in the same network environment and metallb is working fine there.
How to troubleshoot metallb + istio issue? Checked kube-proxy and we have not configured IPVS.

**Comment by danderson** [comment]
Nope, no idea. Other people have working Istio installs with MetalLB, so I suspect Istio is not actually the problem here. Unfortunately, I currently don't have time to help troubleshoot. I suggest tracing the ARP traffic from the client, through the nodes, and into the MetalLB pod. The usual source of problems with layer2 mode are split routing across multiple interfaces, or address conflicts where something other than MetalLB also owns an IP.

**Comment by uablrek** [comment]
Might be the same as; https://github.com/google/metallb/issues/355#issuecomment-450097372

**Comment by cdrage** [comment]
@vijaykumark I'm still getting this issue, did you manage to fix it? I'm running Kubernetes 1.13.2 with Istio + MetalLB and no change.
